src/main/java/org/jvnet/hudson/update_center/Plugin.java

```diff
@@ -254,22 +254,9 @@ public String getScmHost() {
         return labels;
     }
 
-    /** @return {@code true} if the plugin's wiki page has the "plugin-deprecated" label. */
-    public boolean isDeprecated() {
```

Comment:
They'll just need to adapt their code, either if a fork, or if using the update center as a dependency. If they set this up such that their stuff automatically breaks, they've done it wrong.